docs/msiv1_token_revocation.md

```diff
@@ -41,15 +41,19 @@ Steps 5-9 are new and show how the RP propagates the revocation signal.
 ### Explanation:
 1. The client (CX) calls some **Resource** with token **T**.
 2. The resource detects **T** is bad (revoked) and returns **401** + **claims C**.
-3. CX parses **C** and calls **MSAL** with `.WithClaims(C).WithClientCapabilities(cp1)`.
-4. MSAL sees the local cached token is "bad" → triggers a refresh flow.
-5. MSAL calls **MITS** with `xms_cc=cp1&token_sha256_to_refresh=SHA256(T)`.
-6. **MITS** is basically a proxy, forwarding the query to **SFRP**.
-7. **SFRP** uses MSAL again to get a **new** token from eSTS.
+3. CX parses **C** and calls **MSAL** **Client** with `.WithClientCapabilities(cp1)`.
+4. MSAL calls **AcquireToken** with `.WithClaims(C)`.
+5. MSAL sees the local cached token is "bad" → triggers a refresh flow.
```

Comment:
```suggestion
5. MSAL inspects its cache first. If it finds a token, the token is considered to have been revoked. MSAL needs to tell the token issuer about it, so that the token issuer can also ignore it.
```